src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -160,6 +160,8 @@ void SotaUptaneClient::finalizeAfterReboot() {
 data::InstallationResult SotaUptaneClient::PackageInstallSetResult(const Uptane::Target &target) {
   data::InstallationResult result;
   Uptane::EcuSerial ecu_serial = uptane_manifest.getPrimaryEcuSerial();
+  // load the target in installed versions so that it can be re-accessed later
+  storage->saveInstalledVersion(ecu_serial.ToString(), target, InstalledVersionUpdateMode::kNone);
```

Comment:
Is it safe to put that in the `installedVersions` table if it hasn't been installed yet?

Reply:
Cool, just checking. Seems fine, then.